ably/export_test.go

```diff
@@ -106,3 +106,21 @@ func (c *Connection) OnState(ch chan<- State, states ...StateEnum) {
 func (c *Connection) OffState(ch chan<- State, states ...StateEnum) {
 	c.offState(ch, states...)
 }
+
+func (c *Connection) RecoveryKey() string {
+	c.state.Lock()
+	defer c.state.Unlock()
+	return c.recoveryKey
+}
+
+func (c *Connection) RemoveKey() {
```

Comment:
This at least shouldn't be public, but I don't understand why it exists at all. It doesn't seem to be called from anywhere.

Reply:
This is helper in `export_test.go` file they are only used in tests